common/interfaces/flash_interface/flash_api.c

```diff
@@ -94,6 +94,26 @@ static bool _wallet_is_filled(uint8_t index) {
   return false;
 }
 
+bool wallet_is_filled(uint8_t index, wallet_state *state_output) {
+  if (MAX_WALLETS_ALLOWED <= index) {
+    return false;
+  }
+
```

Comment:
Should call `get_flash_ram_instance()` here to load the instance incase not already done.

Reply:
good catch